src/protocol/attribution/accumulate_credit.rs

```diff
@@ -224,31 +229,18 @@ mod tests {
         ];
 
         for x in input {
-            let (h0, (h1, h2)): (Vec<_>, (Vec<_>, Vec<_>)) = x
-                .iter()
-                .map(|y| {
-                    let ss = share(Fp31::from(*y), rng);
-                    (ss[0], (ss[1], ss[2]))
-                })
-                .unzip();
-            shares[0].push(AttributionInputRow {
-                is_trigger_bit: h0[0],
-                helper_bit: h0[1],
-                breakdown_key: h0[2],
-                value: h0[3],
-            });
-            shares[1].push(AttributionInputRow {
-                is_trigger_bit: h1[0],
-                helper_bit: h1[1],
-                breakdown_key: h1[2],
-                value: h1[3],
-            });
-            shares[2].push(AttributionInputRow {
-                is_trigger_bit: h2[0],
-                helper_bit: h2[1],
-                breakdown_key: h2[2],
-                value: h2[3],
-            });
+            let itb = share(Fp31::from(x[0]), rng);
```

Comment:
I couldn't understand the original very well; hopefully this reads a little better.